src/main/java/org/killbill/billing/plugin/payment/dao/PaymentTestDao.java

```diff
@@ -38,6 +39,7 @@
 import org.killbill.billing.plugin.payment.dao.gen.tables.records.TestpaymentPaymentMethodsRecord;
 import org.killbill.billing.plugin.payment.dao.gen.tables.records.TestpaymentResponsesRecord;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.google.common.collect.ImmutableList;
```

Comment:
We're slowly moving away from Guava. You could use `List.of` here for instance instead. /cc @xsalefter

Reply:
... to be precise, `Collections.emptyList()` as seems like you want to pass empty list.